Update `Nomination_Assessment_Team` wiki article (#8913)
ppy/osu-wiki: wiki/People/The_Team/Nomination_Assessment_Team/en.md

Before adding a new advisor, the addition is sanity checked with the rest of the NAT in case of objections. If no issues arise, the new advisor will then be subject to activity expectations depending on their specific situation. If the contribution of an advisor is considered lacklustre or unnecessary at some later point, again as judged by the designated member, then they will be moved back to being an evaluator.

## NAT Leadership

> As of March 2023, the NAT has decided to re-adopt the concept of [leadership](/wiki/Modding/QAT_Leaders), which was previously used in the [QAT](/wiki/Modding/Quality_Assurance_Team). NAT leaders have the combined responsibility of both evaluators and advisors so they can watch over and stay involved with every aspect of the NAT.

Reviewer: ```suggestion
As of March 2023, the NAT has decided to re-adopt the concept of leadership, which was [previously used](/wiki/Modding/QAT_Leaders) in the [QAT](/wiki/Modding/Quality_Assurance_Team). NAT leaders have the combined responsibility of both evaluators and advisors so they can watch over and stay involved with every aspect of the NAT.
```

> The current NAT leaders are ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) and ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099).

Reviewer: <img src="https://user-images.githubusercontent.com/36758269/223821331-0561cd9f-a058-4b97-9b01-ef27ffa33ee8.png" height=250/>

Author: 🔥

### Responsibilities

NAT leaders aim to cover the following tasks:

> - **Activity management:** Constantly monitoring the activity of other NAT members and handling cases of low activity, either through regular checkups or through exit interviews, in case the activity issues of a certain member are problematic.

Reviewer: ```suggestion
- **Activity management:** Constantly monitoring the activity of other NAT members and handling cases of low activity, either through regular checkups or through exit interviews, in case case of activity issues regarding certain members.
```

- **Communication:**

>   - Acting as a bridge between the osu!team, the NAT, and the mapping community, constantly communicating any changes/concerns/demands raised by either of those three parties.

Reviewer: ```suggestion
  - Acting as a bridge between the osu! team, the NAT, and the mapping community, constantly communicating any changes, concerns, or demands raised.
```

>   - Encourage participation in internal and external discussions by proactively engaging all parties, keeping discussions organized, and facilitating the combination of different viewpoints to form a practical course of action.

Reviewer: ```suggestion
  - Encourage participation in internal and external discussions by proactively engaging all parties, keeping discussions organised, and facilitating the combination of different viewpoints to form a practical course of action.
```

| ::{ flag=DE }:: [Capu](https://osu.ppy.sh/users/2474015) | German | Evaluator |
| ::{ flag=GB }:: [Dusk-](https://osu.ppy.sh/users/6092181) | Urdu, some Arabic | Evaluator |
| ::{ flag=HK }:: [Faputa](https://osu.ppy.sh/users/845733) | Cantonese, Chinese | Evaluator |
| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | Arabic, French, some Italian | Leader |

> | ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) |  | Leader |

Reviewer: `meta/group-info` needs updating @cl8n i think

although i have something in the works to replace it very soon
